### DISCUSSION TITLE
May I know the schedule of official version for 3.0+ ?

### DISCUSSION THREAD

**Discussion by Chris2137** [hm21/pro_image_editor#63] [Q&A] [answered]
Hi @hm21, may I know when can we expect the 3.0 release ? Have been looking forward to the feature to edit image inline (without opening up dialog).

Thanks again for the great work.

**Comment by hm21** [comment, marked as answer]
I do not have a fix release date planned, but there is only one last open task that I need to finish, thereafter, I can release it. That open task is an issue I need to resolve where layers are not correctly changing the transformation position when we change the crop area size.
Anyway, with a bit of luck, if I can resolve the issue quickly I can possibly release it this week.

**Reply by Chris2137** [reply]
Thanks @hm21 for the info. Look forward to it.
